tensorflow/core/kernels/data/stats_utils.cc

```diff
@@ -14,6 +14,7 @@ limitations under the License.
 ==============================================================================*/
 #include "tensorflow/core/kernels/data/stats_utils.h"
 
+#include "absl/base/attributes.h"
```

Comment:
Internally, I am seeing an error that indicates that `tensorflow/core/kernels/data:stats_utils` does not depend on a module exporting `third_party/absl/base/attributes.h`. Could you please fix that? Thanks.

Reply:
Should be done.